src/sentry/api/bases/organization_events.py

```diff
@@ -147,9 +110,6 @@ def get_snuba_query_args_legacy(self, request, organization):
         except InvalidSearchQuery as exc:
             raise OrganizationEventsError(exc.message)
 
-        # Filter out special aggregates.
-        self._filter_unspecified_special_fields_in_conditions(snuba_args, set())
```

Comment:
This was removed as we won't be supporting filtering on aggregate columns in the short term, and with fully dynamic aggregate columns we would need expand the query parser to accomodate aggregate filtering.

Reply:
Can we go ahead and remove the function completely?